src/test/java/com/kodekonveyor/cdd/notintegrated/ContractCreationServiceContract.java

```diff
@@ -22,59 +20,35 @@
 })
 public class ContractCreationServiceContract {
 
-  @Autowired
-  ContractInfoFactory<ExampleService> contractInfoFactory;
-
   @Subject
   @InjectMocks
   public ContractCreationServiceImpl<
       ExampleService> contractCreationServiceImpl;
 
   @ContractFactory
   public ContractInfo<ContractCreationServiceImpl<
-      ExampleService>> it;
+      ExampleService>> it; //NOPMD ShortVariable
```

Comment:
I respect Stephen King, but we should use more chatty names than him.

Reply:
This is a very often used name. Other test frameworks also use 'it' for that purpose.
See the contracts for its usage.